src/swe/swe_ceed_impl.h

```diff
@@ -141,11 +64,15 @@ CEED_QFUNCTION(SWEFlux_Roe)(void *ctx, CeedInt Q, const CeedScalar *const in[],
   const CeedScalar gravity = context->gravity;
 
   for (CeedInt i = 0; i < Q; i++) {
-    SWEState   qL = {q_L[0][i], q_L[1][i], q_L[2][i]};
-    SWEState   qR = {q_R[0][i], q_R[1][i], q_R[2][i]};
+    SWEState   qL = {.h = q_L[0][i], .hu = q_L[1][i], .hv = q_L[2][i]};
+    SWEState   qR = {.h = q_R[0][i], .hu = q_R[1][i], .hv = q_R[2][i]};
     CeedScalar flux[3], amax;
     if (qL.h > tiny_h || qR.h > tiny_h) {
-      SWERiemannFlux_Roe(gravity, tiny_h, h_anuga, qL, qR, geom[0][i], geom[1][i], flux, &amax);
+      switch (flux_type) {
```

Comment:
@jeff-cohere, @jedbrown, Do we have to compile the code differently for compiler inlining to be turned on?

Reply:
Not to my knowledge, but I'll leave this Q to the expert. :-)

Reply:
The `CEED_QFUNCTION` and `CEED_QFUNCTION_HELPER` have attributes to aggressively inline.